flippingbitss/react-daterange-picker
# Add storybook

## Motivation

Currently, there is no quick way to experiment with component implementation and see the results. My proposal is to add the storybook.

After dependencies installation the user just needs to run:

```
npm run storybook
```

### How to test

- clone the repository
- run `npm install` -> `npm run storybook`
- The storybook should open with a story containing the date range picker
